build.gradle

```diff
@@ -29,6 +29,7 @@ plugins {
     id 'com.diffplug.spotless' version '6.7.1'
     id 'com.github.hierynomus.license' version '0.16.1'
     id 'com.github.spotbugs' version '5.0.6'
+    id 'distribution'
```

Comment:
can we add a comment specifying that this is for compatibility with https://github.com/airbytehq/airbyte/blob/master/build.gradle#L222 and can be removed when we refactor this?